packages/webapp/src/containers/Help/saga.js

```diff
@@ -31,12 +32,26 @@ export function* supportFileUploadSaga({ payload: { file, form } }) {
     const formData = new FormData();
     formData.append('_file_', file);
     formData.append('data', JSON.stringify(form));
-    const result = yield call(axios.post, supportUrl(), formData, {
-      headers: {
-        'Content-Type': 'multipart/form-data',
-        Authorization: 'Bearer ' + localStorage.getItem('id_token'),
-      },
-    });
+    let result;
```

Comment:
Because I could not get the enum to update in the database, I decided to make this an either or situation